src/poliastro/plotting.py

```diff
@@ -52,11 +52,29 @@ def __init__(self, ax=None, num_points=100):
         num_points : int, optional
             Number of points to use in plots, default to 100.
 
+        bgcolor : tuple or string, optional
```

Comment:
The `Axes` background color is easy to change from the outside, but I admit the rest of the properties are not. Also we cannot just pass the `**kwargs` since we're performing two different kind of plots. I would remove this parameter to simplify the function, but leave the other two.